content/en/docs/faq/devops/install-jenkins-plugins.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Install Plugins to Jenkins in KubeSphere"
+keywords: "KubeSphere, Kubernetes, DevOps, Jenkins, Plugin"
+description: "How to install plugins to Jenkins in KubeSphere"
+linkTitle: "Install Plugins to Jenkins in KubeSphere"
+Weight: 16810
+---
+
+The KubeSphere DevOps System offers containerized CI/CD functions based on Jenkins. The primary means of enhancing the functionality of Jenkins is to install plugins. This tutorial demonstrates how to install plugins on the Jenkins dashboard.
+
+## Prerequisites
+
+You need to enable [the KubeSphere DevOps system](../../../pluggable-components/devops/).
```

Comment:
Not all Jenkins plugins have good maintaining. I would suggest that we provide some suggestions here to avoid miss leading our users. Even worse, some plugins might cause KubeSphere panic.

Reply:
I added a notice/warning in line 11-15. Please help review it, thanks.